data-client/src/main/java/org/wikipedia/gallery/ImageInfo.java

```diff
@@ -39,7 +39,7 @@
     /**
      * thresholdHeight, The minimum height of the image in px.
      */
-    final private int thresholdHeight = 220;
+    final private static int thresholdHeight = 220;
```

Comment:
The convention for constants is uppercase: `THRESHOLD_HEIGHT`
This is worth a read: https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html :-)